index.css

```diff
@@ -78,7 +78,7 @@ body {
 }
 
 .Tym {
```

Comment:
Always use lowercase whenever you are assigning a class in HTML and CSS.

Reply:
noted and taken 
thank you